### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by garrettheel** [PyGithub/PyGithub#430]
Hi, can you please publish a new release. `add_membership` is currently broken in the latest release due to this changeset not being included:

https://github.com/PyGithub/PyGithub/commit/a06b5ecfb215eda49c63d06a49cfefb33b12ad9e#diff-20d5f1aeabe90232d524e590cc5d1972

**Comment by babineaum** [comment]
👍

**Comment by babineaum** [comment]
I would love to have this cut off master, or wait till we can get a few of the smaller PRs in.

**Comment by jakirkham** [comment]
Dup of issue ( https://github.com/PyGithub/PyGithub/issues/391 ).

**Comment by babineaum** [comment]
Great that its a dupe... Maybe if we will the issues up we can get a release?

**Comment by jzelinskie** [comment]
Hey,

I'd love to get another release out, but the build appears to be broken by the last commit.
@jayfk is there an ETA on a fix for this? Next time you should really open a pull request, so that the head of master is always able to be deployed.

**Comment by jayfk** [comment]
@jzelinskie Sorry, I've tested it locally but forgot to add the changes to the replay data to the commit. For whatever reason they weren't serializable. 

Fixed in https://github.com/PyGithub/PyGithub/commit/9134aa2b0454f71ab2baf8a99b12b23d47c6be69, build is green again.

Have you got a chance to think about my comment in https://github.com/PyGithub/PyGithub/issues/391#issuecomment-234951469?

It's a bit of more work right now, but saves you a lot of time for each new release in the future.
